### DISCUSSION TITLE
Help setting up motion detection to record based on lighting

### DISCUSSION THREAD

**Discussion by AustinLG** [gopro/labs#663] [Q&A]
Hello, 

I am currently using a Hero11 Black with the most recent labs firmware installed. I am looking to have the camera start recording a 5-minute video anytime it detects the scene goes from very dark (virtually black) to very bright (well lit). It appears the scene detection does this just fine and registers the sudden transition from dark to light as motion. 

My problem is that the motion detection also treats the transition from light to dark as "motion" and always records a second video when my timed lights have turned back off. Is there any way of disabling this? Or setting a period of time after a trigger where motion is ignored? If so then I could just turn my lights off before that window of time is up. 

Thank you, I've experimented just about as much as I can, and cannot seem to nail this. 

Best,
Austin

**Comment by dnewman-gpsw** [comment]
seems like you should be using a more complex visual script like this:

![image](https://github.com/gopro/labs/assets/10856084/c3be683d-e1ee-43b4-9dc1-a628285b53c3)

using https://gopro.github.io/labs/build/

![Record Bright Transitions-12_5_2023, 08_59_02](https://github.com/gopro/labs/assets/10856084/27d91ff1-821e-4147-8418-5bcd51f9f0a4)

You can enhance/edit this script, by loading the QR Code PNG via the "import Image" button.
![image](https://github.com/gopro/labs/assets/10856084/6ade4a93-d2d9-48f1-a2c3-abb37d3b2fb7)
